examples/connect-to-cluster.js

```diff
@@ -0,0 +1,32 @@
+//This is an example script to connect to a running cluster.
+// After connecting to the cluster I am attempting to set and get a value.
+
+//To setup this cluster you can follow the guide here : 
+// https://redis.io/docs/manual/scaling/
+// In this guide the ports which are being used are 7000 - 7005 
```

Comment:
```suggestion
```

Removed as not needed.

Reply:
I felt it was important to mention the port numbers which I am using since the master nodes in the example script are using the ports 7001 - 7003.